com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -483,12 +483,19 @@ public override void OnInspectorGUI()
             DrawRenderingSettings(camType, rpAsset);
             DrawEnvironmentSettings(camType);
 
-            // Settings only relevant to base cameras
             if (camType == CameraRenderType.Base)
             {
+                // Settings only relevant to base cameras
                 DrawOutputSettings(rpAsset);
                 DrawStackSettings();
             }
+            else if (camType == CameraRenderType.Overlay)
+            {
+                // ScriptableRenderContext.SetupCameraProperties still depends on camera target texture
+                // In order for overlay camera not to override base camera target texture we null it here
+                if (settings.targetTexture.objectReferenceValue != null)
```

Comment:
This is probably ok for upgrade reasons. However we should make sure the change gets applied when the user change from base -> overlay in UI and that we properly handle Undo in that case, restoring the camera target texture.

Reply:
Fixed in latest. Nice catch!